src/plugins/intel_gpu/include/intel_gpu/runtime/utils.hpp

```diff
@@ -173,6 +173,16 @@ inline bool any_not_zero(const std::vector<T> vec) {
     return std::any_of(vec.begin(), vec.end(), [](const T& val) { return val != 0; });
 }
 
+template <typename T, typename P>
```

Comment:
just a question: why do we have separate type of `T` and `P`? Aren't they same? @sshlyapn

Reply:
Looks like a copy-paste from onednn. 
Maybe just use stl based imp insteadl?

```cpp
template <typename T>
inline bool one_of(const T& val, const std::vector<T>& vec) {
    return std::any_of(vec.begin(), vec.end(), [&val](const T& v) { return v == val; });
}
```

Reply:
Applied